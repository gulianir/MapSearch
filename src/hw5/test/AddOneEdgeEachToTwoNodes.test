# Create a graph
CreateGraph testGraph

# Add 1 node
AddNode testGraph Node1

# Add 1 more node
AddNode testGraph Node2

# Add 1 edge to each node
AddEdge testGraph Node1 Node2 Edge1
AddEdge testGraph Node2 Node1 Edge2