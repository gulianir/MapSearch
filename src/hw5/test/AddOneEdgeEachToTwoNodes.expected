# Create a graph
created graph testGraph

# Add 1 node
added node Node1 to testGraph

# Add 1 more node
added node Node2 to testGraph

# Add 1 edge to each node
added edge Edge1 from Node1 to Node2 in testGraph
added edge Edge2 from Node2 to Node1 in testGraph